### ISSUE TITLE
Audio capture from a specific application

### ISSUE THREAD

**Issue by NicolasERT** [naver/prismlivestudio#11]
First of all, congratulations on the great work on this project.
These days I have found a new obs plugin (still in beta) that can be used to capture specific application audio.

https://obsproject.com/forum/resources/win-capture-audio.1338/

I would like to know if you are planning to incorporate similar functionality here. Thanks!

**Comment by zengvain** [comment]
Yes, we are planed to support such obs plugin in coming version. Maybe inside the 1st half year of 2022

**Comment by linuxmobile** [comment]
we are waiting...
